src/neo/SmartContract/Native/NeoToken.cs

```diff
@@ -301,6 +301,12 @@ public UInt160 GetCommitteeAddress(StoreView snapshot)
 
         private CachedCommittee GetCommitteeFromCache(StoreView snapshot)
         {
+            if (snapshot.GetHeader(snapshot.CurrentBlockHash).Timestamp < TimeProvider.Current.UtcNow.AddDays(-2).ToTimestampMS())
```

Comment:
Hi Shargon. Just repeating my feedback: I think 2 days is too much.  Why not use the view number as you mentioned?

Reply:
Because `ViewNumber` is p2p level, we doesn't have this on `onPersist` or on `onVerify`

Reply:
Maybe similiar logic can be applied to validators too. And 2 days period might be somehow too long..

Reply:
What do you think @erikzhang?

Reply:
I think It may not solve the problem. In the next committee epoch ( = 21 blocks), it'll go back to the old set of committees.